test/automated/ansible/roles/create-inventory/tasks/main.yaml

```diff
@@ -9,7 +9,7 @@
 
 - name: create inventory
   template:
-    src: "{{ output_inventory }}.j2"
+    src: "inventory.ec2.j2"
```

Comment:
in macos we are using `inventory.macos.ec2` Aren't we loosing this with this change?

Reply:
Given that we have a platform in the folder path, should not be a problem for CI, but for local runs, we will need to slightly adjust it.

Reply:
But this is the source. For macos the source should be `inventory.macos.ec2.j2` as the templates are different. Or am I loosing something?

Reply:
You are right, I will think about how to split it.